src/Module/BaseApi.php

```diff
@@ -238,7 +238,7 @@ protected static function getCurrentApplication()
 	 *
 	 * @return int User ID
 	 */
-	protected static function getCurrentUserID()
+	public static function getCurrentUserID(bool $nologin = false)
```

Comment:
I'd prefer if we created a separate method with a meaningful name instead of creating a conditional flag to an existing method. The reason is that the meaning is hidden by the flag (that isn't even documented in your changes) and the method becomes overly broad to be intuitive.

I'll be creating a PR to show what I mean.

Reply:
See https://github.com/friendica/friendica/pull/10371